### PR TITLE
[ui] Whitelist: {ethereum,sifchain}.mainnet: rm TRB, MKR, fix GRT address

### PR DESCRIPTION
**Pull request by 59023g**
Remove TRB, MKR, fix GRT address. This is supposed to be final cut.

### CODE REVIEW

**Review by utx0 — DISMISSED** [review]

File: ui/core/src/assets.ethereum.mainnet.json [hunk 1]
```diff
@@ -259,7 +239,7 @@
       "transferLimit": "460405156537753272320"
     },
     {
-      "address": "0xb83cd8d39462b761bb0092437d38b37812dd80a2",
+      "address": "0xc944e90c64b2c07662a292be6244bdf05cda44a7",
```
Comment (ElliotFriedman) [inline ui/core/src/assets.ethereum.mainnet.json:242]: Why was this address changed?

Comment (59023g) [reply ui/core/src/assets.ethereum.mainnet.json:242]: See Sifchain Launch Token Google Doc: https://docs.google.com/spreadsheets/d/12yVNc7VCNdR3hzWsr5NJhfxBGRAduUsLSeYpVmwN-qg/edit#gid=0&range=D19

Comment (utx0) [reply ui/core/src/assets.ethereum.mainnet.json:242]: +1

Comment (ElliotFriedman) [reply ui/core/src/assets.ethereum.mainnet.json:242]: Where did we get that grt token address from? Was that entered into the spreadsheet like that?

Comment (59023g) [reply ui/core/src/assets.ethereum.mainnet.json:242]: GRT https://etherscan.io/token/0xc944e90c64b2c07662a292be6244bdf05cda44a7

Comment (59023g) [reply ui/core/src/assets.ethereum.mainnet.json:242]: That's how it was this morning when I checked over everything

**Review by ElliotFriedman — APPROVED** [review]

**Pull request merged**